libbeat/processors/actions/add_labels.go

```diff
@@ -26,66 +26,103 @@ import (
 	"github.com/elastic/beats/libbeat/processors"
 )
 
-type addLabels struct {
-	labels common.MapStr
+type addFields struct {
```

Comment:
Why do you mix the two in the same file? I see the dependency but shouldn't we split it up into 2 files for clarity?